propagation/trace_context.go

```diff
@@ -63,6 +64,11 @@ func (tc TraceContext) Inject(ctx context.Context, carrier TextMapCarrier) {
 		sc.SpanID(),
 		flags)
 	carrier.Set(traceparentHeader, h)
+
+	bStr := baggage.FromContext(ctx).String()
```

Comment:
Thanks @Aneurysm9 for letting me know, if no changes are required then can you please close this issue ?
https://github.com/open-telemetry/opentelemetry-go/issues/2793

Reply:
I'm not sure that the issue is invalid.  This attempt to fix it, however, is.

Reply:
@Aneurysm9 
I wanted you to close the issue, but i think you have mistakenly closed my PR. If my changes doesn't looks good to you then i can try different approach to work on this issue.

I would request you to please open this PR, so i could start working on this issue again. And it will be really helpful for me, if you could please confirm whether this is a valid issue(https://github.com/open-telemetry/opentelemetry-go/issues/2793) or not ??

Reply:
I intended to close this PR.  This approach is not viable.  Please open a new PR with a new approach if you would like to try again.  

The linked issue appears to be valid from the user reports, but I have not personally validated it.  If you present a PR that fixes it in an appropriate manner and includes tests that demonstrate it has been fixed then the issue can be resolved when that PR is merged.  Alternately, a PR with tests demonstrating that the functionality is working as expected would also suffice.